ci/ci/build.py

```diff
@@ -810,6 +810,20 @@ def __init__(self, params, database_name, namespace):
         self.admin_secret_name = f'sql-{self._name}-{self.admin_username}-config'
         self.user_secret_name = f'sql-{self._name}-{self.user_username}-config'
 
+        self.volumes = [{
```

Comment:
This needs a rebase.  There is no more volumes, there is secrets that specify a name, namespace and mount path.